.github/renovate.json5

```diff
@@ -10,8 +10,7 @@
   automerge: true,
   baseBranches: [
     'main',
-    'release-0.32',
-    'release-0.33',
```

Comment:
This was done to only support two releases; to,herwise, renovate will try to upgrade all branches matching the regex

Reply:
Yeah but i noticed that the old branches are deleted after new releases are created. So if the next release is 0.35, then it makes sense to have a 0.34, and 0.33, so then 0.32 is deleted without needing to modify renovate configs